server/resolvers/findOrCreateTestPlanReportResolver.js

```diff
@@ -11,7 +11,7 @@ const findOrCreateTestPlanReportResolver = async (_, { input }, context) => {
     }
 
     const [testPlanReport, createdLocationsOfData] =
-        await getOrCreateTestPlanReport(input, { status: 'DRAFT' });
+        await getOrCreateTestPlanReport(input);
```

Comment:
Something that concerns me is that there is no way, even through the API, to move a report back to the queue after it's been marked final. Considering history, I can remember instances of wanting to pull content back to the queue.

I think there are multiple potential ways to solve this. One would be to add a special resolver to the testPlanReportOperations which would remove the approvedAt date. This would allow us to use the API to move content back to the queue.

Another solution could be here. It's probably too subtle, and should probably be documented with a comment, but in the past the status: "DRAFT" code above would move a report back to the queue if it already existed. That has come in handy multiple times. The equivalent with approvedAt would be setting approvedAt to null here.

Reply:
> Something that concerns me is that there is no way, even through the API, to move a report back to the queue after it's been marked final. Considering history, I can remember instances of wanting to pull content back to the queue.
> 
> I think there are multiple potential ways to solve this. One would be to add a special resolver to the testPlanReportOperations which would remove the approvedAt date. This would allow us to use the API to move content back to the queue.

Good idea. I've pushed https://github.com/w3c/aria-at-app/pull/715/commits/9016c45f1e29c6d76ac669b6672d1ff59e3cdf87 which adds a new resolver to do that action

> Another solution could be here. It's probably too subtle, and should probably be documented with a comment, but in the past the status: "DRAFT" code above would move a report back to the queue if it already existed. That has come in handy multiple times. The equivalent with approvedAt would be setting approvedAt to null here.

I've already gone with a special resolver in this instance but also related and of note, another (and much more destructive operation here) is that when updating a TestPlanVersion's phase to "DRAFT" it will also nullify all the `approvedAt` properties for the connected `TestPlanReport`s

Reply:
Great change! I verified it's working.